package-lock.json

```diff
@@ -1349,7 +1379,6 @@
       "version": "1.1.3",
       "resolved": "https://registry.npmjs.org/define-properties/-/define-properties-1.1.3.tgz",
       "integrity": "sha512-3MqfYKj2lLzdMSf8ZIZE/V+Zuy+BgD6f164e8K2w7dgnpKArBDerGYpM46IYYcjnkdPNMjPk9A6VFB8+3SKlXQ==",
-      "dev": true,
```

Comment:
What are all those `dev` changes ? They seem unrelated.

Reply:
This file is automatically created when a dependency is installed.
There are two types of dependencies:
This is for development and product use.
Developmental use is an ancillary commercial module for development.
Product use is a required module when the product is running.
As sqlite3 was installed, the development was changed for the product.